streamer/src/nonblocking/quic.rs

```diff
@@ -315,14 +315,12 @@ async fn run_server(
         .store(endpoints.len(), Ordering::Relaxed);
     let staked_connection_table: Arc<Mutex<ConnectionTable>> =
         Arc::new(Mutex::new(ConnectionTable::new()));
-    let (sender, receiver) = async_bounded(coalesce_channel_size);
-    tokio::spawn(packet_batch_sender(
-        packet_sender,
-        receiver,
-        exit.clone(),
-        stats.clone(),
-        coalesce,
-    ));
+    let (sender, receiver) = bounded(coalesce_channel_size);
+    let exit_clone = exit.clone();
+    let stats_clone = stats.clone();
+    thread::spawn(move || {
+        packet_batch_sender(packet_sender, receiver, exit_clone, stats_clone, coalesce);
+    });
```

Comment:
There is this cool pattern that prevents clones from polluting the outer scope
```suggestion
    
    thread::spawn({
    let exit = exit.clone();
    let stats = stats.clone();
    move || {
        packet_batch_sender(packet_sender, receiver, exit, stats, coalesce);
    }
    });
```

Reply:
done